src/utils.js

```diff
@@ -0,0 +1,12 @@
+export const cleanBookData = (book) => {
+  return  {
+    id: book.id,
+    isbn: book.isbn,
+    title: book.title,
```

Comment:
It'll be a fun challenge to figure out the logic of how to format this title correctly!